ui/raidboss/data/02-arr/raid/t8.ts

```diff
@@ -112,18 +116,19 @@ const triggerSet: TriggerSet<Data> = {
       },
     },
     {
+      netType: 'StartsUsing',
       id: 'T8 Brainjack',
       netRegex: NetRegexes.startsUsing({ id: '7C3', source: 'The Avatar' }),
       netRegexDe: NetRegexes.startsUsing({ id: '7C3', source: 'Avatar' }),
       netRegexFr: NetRegexes.startsUsing({ id: '7C3', source: 'Bio-Tréant' }),
       netRegexJa: NetRegexes.startsUsing({ id: '7C3', source: 'アバター' }),
       netRegexCn: NetRegexes.startsUsing({ id: '7C3', source: '降世化身' }),
       netRegexKo: NetRegexes.startsUsing({ id: '7C3', source: '아바타' }),
-      alertText: (data, matches: MatchesStartsUsing, output) => {
+      alertText: (data, matches: NetAllMatches['StartsUsing'], output) => {
```

Comment:
Yeah, I talked about this in the initial PR comment, this is 100% auto-inferred now if `netType` is specified, so the logic should probably be `netType is set || <previous logic>`, I'm just not familiar at all with the eslint scripts.

Reply:
Actually, upon further review, the old eslint rule isn't required at all now I think? Since in all cases it's either auto-inferred or irrelevant due to using only canned conditions/responses.

Reply:
I removed the eslint rule with the latest commit, since that's what made sense to me. Let me know if you'd rather it be reverted and modified instead.

Reply:
I am not quite understand what is wrong here. Is it just the lint rule complains but `tsc` says nothing?

Reply:
The lint rule doesn't serve any purpose after this PR, since the `matches` parameter is either always properly typed (if `type: ` property is specified) or irrelevant (if only canned conditions/responses are used) if the trigger is a netRegex, and since all typescript conversions will have to be netRegex per quisquous in another comment on this PR, it'll always be unneeded.

Reply:
To clarify further, consider the following cases:

1. `type` is specified, `matches` is used in a callback. `matches` is automatically properly typed:
![image](https://user-images.githubusercontent.com/6119598/122681154-03f1be80-d1c1-11eb-8488-ef4443aea0db.png)
2. `type` is not specified, trigger functions only use canned responses. `matches` is never specifically in the trigger so the old rule doesn't apply:
![image](https://user-images.githubusercontent.com/6119598/122681202-3ac7d480-d1c1-11eb-95cd-913cdc3352e3.png)
3. Someone attempts to write a trigger that does not specify type but attempts to use a non-canned response. tsc won't compile since it can't discriminate the union properly:
![image](https://user-images.githubusercontent.com/6119598/122681250-6e0a6380-d1c1-11eb-9c31-b4c5241d9038.png)